test/test.js

```diff
@@ -8,18 +8,31 @@ describe('session-resume', function () {
       <form>
         <input id="my-first-field" value="first-field-value" class="js-session-resumable" />
         <input id="my-second-field" value="second-field-value" class="js-session-resumable" />
```

Comment:
I would guess one problem with the tests is that they are looking for `input[type=text]` and these are missing a type

```suggestion
        <input id="my-first-field" type="text" value="first-field-value" class="js-session-resumable" />
        <input id="my-second-field" type="text" value="second-field-value" class="js-session-resumable" />
```